client/base_client.go

```diff
@@ -48,8 +48,7 @@ type baseClient struct {
 	security SecurityOption
 
 	gRPCDialOptions []grpc.DialOption
-
-	timeout time.Duration
+	timeout         time.Duration
 }
```

Comment:
Why change this?

Reply:
This may be caused by the code format automatic specification plugin of vscode - -

Reply:
Because the empty line between `gRPCDialOptions` and `timeout ` is removed, so the fmt will make their types align.